sample-android/app/build.gradle.kts

```diff
@@ -19,19 +15,23 @@ tasks.withType<KotlinCompile> {
 }
 
 dependencies {
-    implementation(Kotlin.stdlibJdk7)
+    implementation(platform(Firebase.bom))
+    implementation(Firebase.authentication)
+    implementation("com.google.firebase:firebase-analytics")
```

Comment:
Maybe we should add this one in the built-in dependencies constants?

Reply:
it's already there. 
Since this platform constraint thing is new and not well-known, I wanted to show explicitely that there is no version